tasks/components/SubtaskTile.tsx

```diff
@@ -47,7 +47,7 @@ export const SubtaskTile = ({
       <div>
         <Checkbox
           onChange={value => onChange({ ...subtask, isDone: value })}
-          checked={subtask.isDone}
+          checked={subtask.isDone !== undefined ? subtask.isDone : false}
```

Comment:
```suggestion
          checked={!!subtask.isDone}
```

Reply:
@florianjs already marked it as resolved?

Reply:
Yes, it just hasn't been on github yet.